roxie/ccd/ccdstate.cpp

```diff
@@ -2588,12 +2592,26 @@ class CRoxiePackageSetManager : implements IRoxieQueryPackageManagerSet, impleme
             }
             else if (strieq(queryName, "control:memlock"))
             {
-                roxiemem::lockRoxieMem(true);
+                int srtn = roxiemem::lockRoxieMem(true);
+                if (!srtn)
+                {
+                    topology->setPropBool("@heapLockMemory", true);
+                    reply.append(" <heapLockMemory locked='1'/>\n");
```

Comment:
Trivial, but typical naming convention has XML tag names starting with capitol, and attribute names starting with lower.

So `<HeapLockMemory locked='1'/>`

Reply:
ok.  I was trying to match the config name option case (heapLockMemory).  Not sure which is better ?

Reply:
In most places in our code I think xml tags are uppercase.  But in roxie control messages there are a few exceptions already, so it's up to you.

Reply:
ok, I think I will leave it as is.